src/cpp/core/include/core/http/TcpIpAsyncClientSsl.hpp

```diff
@@ -70,6 +71,15 @@ class TcpIpAsyncClientSsl
       // use scoped ptr so we can call the constructor after we've configured
       // the ssl::context (immediately above)
       ptrSslStream_.reset(new boost::asio::ssl::stream<boost::asio::ip::tcp::socket>(ioService, sslContext_));
+
+      // Set the SNI so we still work with TLS v1.3
```

Comment:
I think we should note that the TLS v1.3 standard requires the SNI to be set instead of just saying that we will continue to work (the context is important, and very helpful here).

Reply:
Sure thing.